### ISSUE TITLE
Find Symbols / Find Types

### ISSUE THREAD

**Issue by nosami** [OmniSharp/omnisharp-roslyn#12]
Omnisharp v1 currently has separate endpoints for these. They are analagous to Ctrl-T in resharper v8

Maybe consider combining these.

**Comment by mat-mcloughlin** [comment]
Is this done now?

**Comment by jrieken** [comment]
yeah - I think this is https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp/Api/Navigation/OmnisharpController.FindSymbols.cs#L15

**Comment by nosami** [comment]
The old server used to put namespace and class names in the text field along with the symbol name.

 We should probably extend the response with this data so the clients can choose to display how they like.

**Comment by david-driscoll** [comment]
@nosami do we still need this?  Should we add `Namespace` and `ClassName` to `SymbolLocation`?

**Comment by nosami** [comment]
Yeah... we should do this at some point.
